code/modules/antagonists/nukeop/equipment/nuclear_bomb/bee_nuke.dm

```diff
@@ -0,0 +1,79 @@
+/// A fake nuke that actually contains beer.
+/obj/machinery/nuclearbomb/bee
+	name = "rayne corp brand nuclear fission explosive"
```

Comment:
is "rayne corp" supposed to be a proper noun? Capitalize and add \improper if so